tests/syft/lib/python/complex/serde_test.py

```diff
@@ -0,0 +1,32 @@
+# syft absolute
+import syft as sy
+from syft.lib.python.complex import Complex
+from syft.proto.lib.python.complex_pb2 import Complex as Complex_PB
+
+
+def test_serde() -> None:
+    syft_complex = Complex(True)
```

Comment:
change this test to pass in a true complex number (with both terms) instead of a bool

that said - also testing what happens when you pass in a bool is interesting!

Reply:
Thanks, it's changed 👍